sqldelight-idea-plugin/build.gradle

```diff
@@ -104,6 +102,7 @@ dependencies {
   implementation libs.moshi
 
   testImplementation libs.truth
+  testImplementation libs.okio
```

Comment:
no idea how to fix tests without direct dependency because android-intellij-plugin pulls old version

Reply:
I'd leave it out of `libs.versions.toml` and then just add a comment for it here that it's because of a transitive intellij issue